docs/docs/firmware/can-traffic/index.md

```diff
@@ -22,60 +21,52 @@ The message length in a CAN frame can be calculated as a function of the data le
 <figcaption>CAN Bus Frame Diagram. Source: <a href="https://www.csselectronics.com/pages/can-bus-simple-intro-tutorial">CSS Electronics CAN Bus Tutorial</a></figcaption>
 </figure>
 
-Each CAN frame consists of several fields with fixed bit lengths, plus a variable-length data field. Here’s the breakdown:
-
-- **Start of Frame (SOF)**: 1 bit
-- **Identifier (ID)**: 11 bits for standard frames or 29 bits for extended frames
-- **Remote Transmission Request (RTR)**: 1 bit
-- **Control (DLC)**: 6 bits
-- **Data**: Variable length, 8 bits per byte, depending on data length \( x \)
-- **CRC**: 16 bits
-- **ACK**: 2 bits
-- **End of Frame (EOF)**: 7 bits
-
-**Calculation for fixed portion(\(a\))** : (\( 1 + 11 + 1 + 6 + 16 + 2 + 7 = 44\)) bits
+Each CAN frame consists of several fields with fixed bit lengths, plus a variable-length data field. Let \(x\) be the number of bytes. Here’s the breakdown:
 
-**Variable Portion (b)**: The data field adds 8 bits for each byte of data. So, \( b = 8 \).
-
-**Bit Stuffing**: To ensure synchronization, CAN adds around 20% bit stuffing to the frame length. Including bit stuffing, the approximate formulas are:
-
-- **For a Standard Frame**: \( M(x) \approx 44 + 10x \) bits
+|Frame Element| Length(bits)|
+|--------------|------------|
+|Start of Frame (SOF) |     8   |
+|Identifier (ID)|      11     |
+|Remote Transmission Request (RTR)| 1|
+| Control (DLC)| 6|
+|Cyclic Redundancy Check (CRC) | 16 |
+|Acknowledgement (ACK)| 2 |
+|End of Frame (EOF)| 7|
+|Data Length| 8 bits per byte with 20% bit stuffing = 9.6x|
 
 ### Final Formula
 
-A message with data length of \( x \) bytes requires up to \( a + bx \) bits.
+The maximum total frame length is: (\( 1 + 11 + 1 + 6 + 16 + 2 + 7 + 9.6x = 44 + 9.6x\)) bits
```

Comment:
Don't need to put this expression in brackets. Since it's pretty long, put it on its own line by using `\[ \]` instead of `\( \)`. Make sure the order of terms matches the table

Would be good to remind the reader what `x` means here